### PR TITLE
Documentation Review

### PR DESCRIPTION
**Pull request by djbe**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,15 +10,18 @@ _None_
...
-  [#301](https://github.com/SwiftGen/SwiftGenKit/issues/301)
+  [#301](https://github.com/SwiftGen/SwiftGen/issues/301)
 * Disable default value for named template option and ensure that there is a template option.  
   [Liquidsoul](https://github.com/liquidsoul)
   [#283](https://github.com/SwiftGen/SwiftGen/issues/283)
 * Templates are now grouped by subcommand on the filesystem. This is only important if you had custom templates in the `Application Support` directory. To migrate your templates, place them in a subfolder with the name of the subcommand, and remove the prefix of the template filename.
   [David Jennes](https://github.com/djbe)
-  [#304](https://github.com/SwiftGen/SwiftGenKit/issues/304)
+  [#304](https://github.com/SwiftGen/SwiftGen/issues/304)
+# The `images` command has been renamed to `xcassets` to better reflect it's functionality.
```
Comment (AliSoftware) [inline CHANGELOG.md]: s/it's/its/

File: CHANGELOG.md [hunk 2]
```diff
@@ -28,7 +31,7 @@ _None_
 
 * Improved installation instructions in the README.  
   [Olivier Halligon](https://github.com/alisoftware)
-  [#303](https://github.com/SwiftGen/SwiftGenKit/issues/303)
+  [#303](https://github.com/SwiftGen/SwiftGen/issues/303)
```
Comment (AliSoftware) [inline CHANGELOG.md]: We should definitely add a _Danger_ rule to check if the links exist and if it points to SwiftGen (e.g. maybe all links associated with each SwiftGen's CHANGELOG entries should point to an issue or PR in the SwiftGen repo not the other repos… except for the PR before a release when we might reference big changes from other repos and ignore the future Danger warning then)

File: README.md [hunk 3]
```diff
@@ -128,17 +128,18 @@ Or add the path to the `bin` folder to your `$PATH` and invoke `swiftgen` direct
...
-* `swiftgen assets [OPTIONS] DIR`
-* `swiftgen colors [OPTIONS] FILE`
-* `swiftgen fonts [OPTIONS] DIR`
-* `swiftgen storyboards [OPTIONS] DIR`
-* `swiftgen strings [OPTIONS] FILE`
+* `swiftgen colors [OPTIONS] FILE1 …`
+* `swiftgen fonts [OPTIONS] DIR1 …`
+* `swiftgen storyboards [OPTIONS] DIR1 …`
+* `swiftgen strings [OPTIONS] FILE1 …`
+* `swiftgen xcassets [OPTIONS] FILE1 …`
```
Comment (AliSoftware) [inline README.md]: In people's minds, is `xcassets` a FILE or a DIR? 🤔

File: README.md [hunk 4]
```diff
@@ -128,17 +128,18 @@ Or add the path to the `bin` folder to your `$PATH` and invoke `swiftgen` direct
...
 
 Each subcommand has its own option and syntax, but some options are common to all:
 
 * `--output FILE` or `-o FILE`: set the file where to write the generated code. If omitted, the generated code will be printed on `stdout`.
 * `--template NAME` or `-t NAME`: define the Stencil template to use (by name, see [here for more info](https://github.com/SwiftGen/templates) to generate the output.
 * `--templatePath PATH` or `-p PATH`: define the Stencil template to use, using a full path.
+* Each command supports multiple input files (or directories where applicable).
 
 You can use `--help` on `swiftgen` or one of its subcommand to see the detailed usage.
```
Comment (AliSoftware) [inline README.md]: + maybe add something like "You have to provide one and only one template when invoking swiftgen commands. (You should use either `-t` OR `-p`, but not BOTH at the same time"?

File: README.md (hunk 3 above)
Comment (Liquidsoul) [inline README.md]: Maybe use `PATH1.xcassets` instead of `FILE1` ?

Comment (AliSoftware) [inline README.md]: Yeah could be a good idea.
But after thinking about it a bit, it's probably not worth it. We want the `Usage` summary (the 1-line invocation template) to be short and comprehensible.

If people need more info (about if they have to use a DIR containing asset catalogs or the PATH to the assets catalog itself or whatnot), they can use `swiftgen xcassets --help`. So maybe that's not worth bikeshedding over this…

Comment (djbe) [inline README.md]: For Finder, it's a folder, for xcode, it's a file (maybe package).

Crazy idea, but what about `CATALOG1`? (or `CAT1`)

File: CHANGELOG.md (hunk 2 above)
Comment (djbe) [inline CHANGELOG.md]: We can at least check if it's a broken link, and that the issue number matches the link's number (we've made that mistake too).

File: README.md (hunk 3 above)
Comment (AliSoftware) [inline README.md]: Oh I like `CATALOG1` actually, nice

File: CHANGELOG.md (hunk 2 above)
Comment (AliSoftware) [inline CHANGELOG.md]: :+1:

**Pull request merged**